### DISCUSSION TITLE
Contribution in Bitcoin Knots

### DISCUSSION THREAD

**Discussion by ghost** [bitcoinknots/bitcoin#39] [Ideas]
I will be a organizing a meetup in Bangalore, India this month and give presentation about Bitcoin Development. Students from different engineering colleges who are good in at least in one of the programming language and interested in Bitcoin will join the meetup. I want to create a small team of devs from this meetup and start contributing to Knots. Let me know if you plan to involve more devs in this repository, increasing number of users running Knots nodes etc.

Other things that I have in mind:

1. Improve website: https://github.com/bitcoinknots/knots-www/issues/1
2. Create a chatroom and organize weekly meetings to discuss things
3. Improve privacy

**Comment by luke-jr** [comment]
This sounds like a good idea.

**Comment by ghost** [comment]
Meetup was good. More than 100 students participated in it.

Slides: https://docs.google.com/presentation/d/13bDlNr_ZmD89ndLg3BqF77BiqJWputwnW51Tf8JEPhk/edit

Pics: https://oph21gallery.bitcoinmeetup.info/

Other details on website: https://oph21.bitcoinmeetup.info/

Lot of students had basic questions about Bitcoin which is great as they were interested and curious to learn more. However, creating a team of devs to contribute in a full node implementation can take more time. I have shared my contact details, part of few private groups and bitcoin related resources. Hopefully we could make a small team in next few months and maybe I will do more meetups in other colleges.

Thanks to all the organizers, volunteers, x00 and @luke-jr :)

Article in Bitcoin Magazine about Knots and meetup is still pending. I have been following up via email. Maybe it gets published this week. If not, I will write a medium post.

I will continue to contribute and start working on few things in next few days.

**Comment by ghost** [comment]
> Improve website:
Improve website knots-www#1
Create a chatroom and organize weekly meetings to discuss things
Improve privacy

1. PRs related to improving website received no positive response so closed

2. Telegram group was created with 30 students from last bitcoin meetup: https://www.linkedin.com/feed/update/activity:6858517579263164416

3. https://github.com/bitcoinknots/bitcoin/issues/42 got no comments

Created few videos from meetings: https://www.youtube.com/channel/UCLO3SyjX2iebfRGQDo0mnSQ and even tried posting internship ads recently
